pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -7,6 +7,9 @@
 , cudaSupport ? false, nvidia_x11 ? null, cudatoolkit ? null, cudnn ? null
 # Default from ./configure script
 , cudaCapabilities ? [ "3.5" "5.2" ]
+, sse42Support ? false
+, avx2Support ? false
+, fmaSupport ? false
```

Comment:
What happens if this is enabled, but not supported by the hardware? Would it still works?